packages/graphql/lib/src/links/websocket_link/websocket_client.dart

```diff
@@ -422,18 +458,34 @@ class SocketClient {
                   parse(message.toJson()),
                 ));
 
+        dataErrorComplete
+            .where((message) => message is SubscriptionNext)
+            .cast<SubscriptionNext>()
```

Comment:
```suggestion
            .whereType<SubscriptionNext>()
```

Reply:
Fixed in https://github.com/zino-hofmann/graphql-flutter/pull/1060/commits/7d4939fa67fb5c42b46749865909c8812d2fcba0